_posts/2021-02-09-fallstudie-sharingangebote.md

```diff
@@ -0,0 +1,213 @@
+Ich brauche pro Weg mit dem Rad 10 Minuten und kann, wenn es das Modell erlaubt, das Fahrzeug während des Einkaufs freigeben.
+Roller eignen sich nicht, da ich gerne die Einkäufe teilweise in einen Korb legen möchte.
+Wir ignorieren monatliche Grundgebühren.
+
+**Angebotsname**|**Kosten [€]**
+:-----:|:-----:
+Stadtteilauto|3,90
+Wuddi|3,80
+Tretty (ohne Abgabe)|6,40
+Tretty (mit Abgabe)|3,80
+
+Selbst wenn ich also das Fahrrad zwischendurch abgebe (und damit riskiere, dass es mir jemand anders in der Zwischenzeit wegschnappt) ist es nicht günstiger als bequem mit dem Auto zu fahren.
+
+## Fall 2: Der Besuch
+Ich besuche jemanden am Stadtrand.
+Für die Strecke von 8 Kilometern pro Richtung brauche ich mit Rad/Roller jeweils eine halbe Stunde und ich gehe davon aus, dass das Ziel innerhalb des Servicesbereichs liegt (ich also die Fahrzeuge zwischendurch abgeben kann), was nicht unbedingt der Fall ist.
+Insgesamt bin ich 4 Stunden fort.
+Wir ignorieren monatliche Grundgebühren.
+
+**Angebotsname**|**Kosten [€]**
+:-----:|:-----:
+Stadtteilauto|14,68
+Wuddi|14,08
+Tretty (ohne Abgabe)|22,60 (14,99*)
+Tretty (mit Abgabe)|7,40
+Lime (ohne Abgabe)|50,00 (9,99*)
+Lime (mit Abgabe)|14,00 (9,99*)
+Tier (ohne Abgabe)|47,60 (9,99*)
+Tier (mit Abgabe)|13,40 (9,99*)
+Swapfiets|- (16,90*)
+
+\* Die Zahlen in Klammern geben jeweils ein günstigeres Abo-Angebot der Anbieter an. Bei Lime und Tier sind dies Tagespässe, bei Tretty und Swapfiets sogar ein Monatsabo.
+Für Nutzungen, welche über einigen Minuten hinaus gehen, scheinen sich also bereits immer solche Abos zu lohnen.
+Zum Vergleich: Ein Flatrate-Angebot für den ganzen Monat kostet bei Lime 39€, Tier scheint Stand heute kein Monatsangebot zu haben.
+
+## Break Even Point für Abos
+Fragt sich also, ab wann sich genau ein Abo lohnt. Hierfür betrachten wir zunächst nur die Roller/Rad-Anbieter und berechnen, wie viel Zeit wir spontan buchen können, bevor die Kosten die des Abos überschreiten.
+Wir ziehen für die Rechnnung jeweils die Aktivierungsgebühr einmalig ab:
+
+**Angebotsname**|**Spontane Zeit für Kosten eines Abos [h]**
+:-----:|:-----:
+Tretty|2,59
+Lime (Tagespass)|0,75
+Lime (Monatsabo)|3,17
+Tier (Tagespass)|0,79
+
+Die Zahlen für die Elektroroller sind niedriger, als ich es erwartet hätte.
+Für die Einbeziehung der Auto-Anbieter wird die Betrachtung komplizierter, da wir hier zwei Parameter variieren können: Die gebuchte Zeit und die gefahrenen Kilometer. 
+Wir stellen uns die Frage, wie weit/lange wir mit den Autos fahren können, bevor wir die Kosten eines Tretty-Monatsabos von 14,99€ erreicht haben.
+Wir lösen die Gleichung `Kilometerkosten x gefahrene Kilometer + Stundenkosten x gebuchte Zeit = 14,99` und bekommen pro Anbieter eine Kurve.
+
+[Stadtteilauto](https://www.wolframalpha.com/input/?i=0.23y%2B2.75x%3D14.99)
+
+[Wuddi](https://www.wolframalpha.com/input/?i=0.28y%2B2.4x%3D14.99)
+
+[Ergebnis](https://www.desmos.com/calculator/l8anciqj6b)
```

Comment:
Diese Links sind für meinen Geschmack ein wenig alleinstehend ohne Erklärung. Vielleicht kannst du zu jedem noch einen kleinen Satz schreiben und dann in einem `<details></details>` Block zum aufklappen verstecken

Reply:
Gute Idee!